.github/workflows/integration.yml

```diff
@@ -9,21 +9,11 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        secured: ["true", "false"]
-        entry:
-          - { opensearch_version: 1.0.1 }
-          - { opensearch_version: 1.1.0 }
-          - { opensearch_version: 1.2.4 }
-          - { opensearch_version: 1.3.7 }
-          - { opensearch_version: 2.0.1 }
-          - { opensearch_version: 2.1.0 }
-          - { opensearch_version: 2.2.1 }
-          - { opensearch_version: 2.3.0 }
-          - { opensearch_version: 2.4.0 }
-          - { opensearch_version: 2.4.1 }
+        opensearch_version: [ '1.0.1', '1.1.0', '1.2.4', '1.3.7', '2.0.1', '2.1.0', '2.2.1', '2.3.0', '2.4.0', '2.5.0' ]
```

Comment:
Can we also add `2.6.0` given that it was released a few days ago. Would be good to test against that as well.